integration_tests/run_tests_on_vm.sh

```diff
@@ -12,7 +12,7 @@ gcloud compute instances create $instance_name \
   --image-family=ubuntu-1404-lts \
   --image-project=ubuntu-os-cloud \
   --machine-type=n1-standard-2 \
-  --scopes default,userinfo-email,useraccounts-ro,cloud-platform \
+  --scopes default,userinfo-email,cloud-platform \
```

Comment:
@bendory said that was needed to read service account token from instance metadata ?

Reply:
Hrm, I've searched internally/externally and that flag seems to no longer exist / has been deprecated. I will check what replaces it. 

Here's the error I was getting:

    Step #2: ERROR: (gcloud.compute.instances.create) Could not fetch resource:
    Step #2: - One or more of the service account scopes are invalid: 'useraccounts-ro'

Reply:
Ok